test/ssl-tests/04-client_auth.conf.in

```diff
@@ -26,15 +26,23 @@ sub generate_tests() {
     foreach (0..$#protocols) {
         my $protocol = $protocols[$_];
         my $protocol_name = $protocol || "flex";
+        my $caalert;
         if (!$is_disabled[$_]) {
+            if ($protocol_name eq "SSLv3") {
```

Comment:
I think this is not worth addressing within the scope of this MR. Like you say I suspect other stuff breaks in that case...and I'm not sure if we care about it anyway.